src/locales/en.json

```diff
@@ -27,7 +27,7 @@
           "show": "Show",
           "required": "This field is required",
           "errors": {
-            "403": "Your password doesn't seem to be correct",
+            "passphrase": "Your password doesn't seem to be correct",
```

Comment:
question & nit: now that we are sure about the error type (i.e. "invalid password"). Should we continue to use "doesn't seem to be" instead of "is"?

Reply:
oui c'était le message initial pour la stack, qui était sûre elle aussi.